db/seeds.rb

```diff
@@ -1,6 +1,7 @@
-User.create( email: "ben@ben.com", encrypted_password: "qqqqqq")
+User.create!( email: "ben@ben.com", password: "qqqqqq")
```

Comment:
would encourage use of puts, even if for test (I'll also add to style guide)

Reply:
on it